app/assets/javascripts/comments.js

```diff
@@ -67,13 +67,15 @@ $(() => {
     const $commentBody = $comment.find('.comment--body');
     const commentId = $comment.attr('data-id');
     const originalComment = $commentBody.clone();
+    const $counter = $tgt.attr('data-body');
 
     const resp = await fetch(`/comments/${commentId}`, {
       credentials: 'include',
       headers: { 'Accept': 'application/json' }
     });
     const data = await resp.json();
     const content = data.content;
```

Comment:
Here's where we get the comment content. It's fetched on line 71, the `await fetch`, and pulled out of the response data into the `content` variable here.